test/test_backend/routes/tst_graph.py

```diff
@@ -32,70 +38,100 @@
 single small,single-small,1000002363,30
 single small,single-small-second-time,1000002363,30
 """
-
 TEST_CASES_HOLD = """
 testType,testName,codeset_ids,timeoutSeconds
 """
-
 TEST_CASES = TEST_CASES_FAST
+EXTRA_CONCEPT_IDS = []
+HIDE_VOCABS = ['RxNorm Extension']
+CONNECTED_SUBGRAPH_FILENAME_PATTERN = 'get_connected_subgraph__{}.json'
 
-csv_file = StringIO(TEST_CASES_FAST.strip())
-reader = csv.DictReader(csv_file, delimiter=',', quotechar='"')
-test_cases_list = list(reader)
-
-extra_concept_ids = []
-hide_vocabs = ['RxNorm Extension']
-
-results_from = {
-    'date': '2023-12-18',
-    'commit': '660394d',
-}
-results = {
-    'get_connected_subgraph': {
-        'single-small': {
-            'results': [[[442793, 4034964], [4129519, 43531010]],
```

Comment:
## Siggie's original results very different
The results saved in Python code that you gave me originally turned out to be very different (far fewer nodes) than what I got when I ran. I'm thinking what you had there was outdated and simply there as placeholder.

I created a fresh `networkx` graph, so I don't think it was a problem on my end.

Another discrepancy in your code (I think because it was just unfinished), was that your 'results' object here had 5 items in its list. But the JSON string you were saving only had 4 items. You had also set it up to expect only 4 objects returned from `get_connected_subgraph()`, but it was returning 5 items.